client/http/interface.go

```diff
@@ -768,12 +770,25 @@ func (c *client) GetSchedulers(ctx context.Context) ([]string, error) {
 	return schedulers, nil
 }
 
+// GetSchedulerConfig returns the configuration of the specified scheduler for pd cluster
+func (c *client) GetSchedulerConfig(ctx context.Context, name string) (map[string]any, error) {
+	var config map[string]any
+	uri := path.Join(SchedulerConfig, name, "list")
```

Comment:
Please create a function in `http/api.go` to generate the URI, such as `SchedulerConfigByName`.